File: 23_C#Basics/_04_LoopsPartOneMoreExercises/_09_Clock/_09_Clock.cs
using System;

public class _09_Clock
{
    public static void Main(string[] args)
    {
        // Print time
        for (int i = 0; i < 24; i++)
        {
            for (int j = 0; j < 60; j++)
            {
                Console.WriteLine($"{i} : {j}");
            }
        }
    }
}

File: 01_ProgrammingBasics/_04_LoopsPartOneMoreExercises/_09_Clock.java
package _01_ProgrammingBasics._04_LoopsPartOneMoreExercises;

public class _09_Clock {
    public static void main(String[] args) {
        // Print time
        for (int i = 0; i < 24; i++) {
            for (int j = 0; j < 60; j++) {
                System.out.printf("%d : %d%n", i, j);
            }
        }
    }
}
